config.yaml

```diff
@@ -21,3 +21,22 @@ options:
       Supported values for this option are
       "*" - Enable RGW on all storage nodes
       ""  - Disable RGW on all storage nodes
+  region:
+    default: RegionOne
+    description: Name of the OpenStack region
+    type: string
+  namespace-tenants:
```

Comment:
Nomenclature question, would "project" not be the more modern term instead of "tenant" in OpenStack?

Reply:
Done